### ISSUE TITLE
The down arrow caret image in the Select control should be aria-hidden or role:none since it is decoration

### ISSUE THREAD

**Issue by camsmithmicrosoft** [microsoft/fast-dna#1915]
Using a screen reader and items navigation, scan through the unopened select. You will note that the screen reader will scan to the caret icon and just say "image"

**Comment by chrisdholt** [comment]
Prefer the implementation of aria-hidden here rather than a role of none or presentation. Thanks @camsmithmicrosoft
